scripts/spo-get-list-library-permission-export-to-csv/README.md

```diff
@@ -133,6 +133,80 @@ StartProcessing
 
 ```
 [!INCLUDE [More about PnP PowerShell](../../docfx/includes/MORE-PNPPS.md)]
+
+# [CLI for Microsoft 365](#tab/cli-m365-ps)
+```powershell
+# Usage example:
+# .\Exoprt-ListLibraryPermissions.ps1 -WebUrl "https://contoso.sharepoint.com/sites/Intranet" -ListName "Demo List"
+
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $true, HelpMessage = "Please enter Site URL, e.g. https://contoso.sharepoint.com/sites/Intranet")]
+    [string]$WebUrl,
+    [Parameter(Mandatory = $true, HelpMessage = "Please enter list title")]
+    [string]$ListName
+)
+begin {
+    #Log in to Microsoft 365
+    Write-Host "Connecting to Tenant" -f Yellow
+
+    $m365Status = m365 status
+    if ($m365Status -eq "Logged Out") {
+        m365 login
+    }
+
+    Write-Host "Connection Successful!" -f Green 
+}
+process {
+    $listPermissions = @()
+    $dateTime = "{0:MM_dd_yy}_{0:HH_mm_ss}" -f (Get-Date)
+    $csvPath = "$($ListName -replace '\s','_')-permissions-" + $dateTime + ".csv"
+
+    # Gets information about the specific list
+    $listInfo = m365 spo list get --webUrl $WebUrl --title $ListName --withPermissions | ConvertFrom-Json
+
+    # Check if list has unique role permissions
+    $hasUniquePermissions = $listInfo.HasUniqueRoleAssignments
+
+    ForEach ($roleAssignment in $listInfo.RoleAssignments) {
+        $permissionType = $roleAssignment.Member.PrincipalType
+        $permissionLevels = $RoleAssignment.RoleDefinitionBindings | Select-Object -ExpandProperty Name
+
+        If ($permissionType -eq 8) {
+            # Get members of a SharePoint Group
```

Comment:
this condition to check if it is `8` is very specific. It's good we have this little comment here 👍 Thank you for that 🙏
Actually it is because the `SharePointGroup` enum member has 8 value assigned to it 
![image](https://user-images.githubusercontent.com/58668583/171759329-9277ccc5-32e0-4d5f-b5ff-31bdae432161.png)
TBH it's a shame that we don't translate it in the CLI response to enum names 🤔 I suppose it may not be that easy 😜. Do you think this may be a good use case to open and issue for a improvement to the CLI for M365 and lets check if this could be done 🤔?

Reply:
Thank you @Adam-it for your review.
The issue with possible resolution is created https://github.com/pnp/cli-microsoft365/issues/3379